docs/Developers/Newcomers_Guide.rst

```diff
@@ -152,7 +184,17 @@ Now you can start working on it.
 
         cobot assign <issue_link>
 
-    Take care to write the full link to the issue
+    Take care to write the full link to the issue.
+    Also takeup the issue, only when you know what the problem is and
+    how to solve it.
+
+    You can do amazing stuff using cobot.
+
+        * Issue assigning as stated earlier.
+        * File issues::
+
+            cobot file issue <repo> <title>
+            <description>
```

Comment:
The most important feature of cobot is `cobot help` which displays info about all other commands. Please at least add this to the list.